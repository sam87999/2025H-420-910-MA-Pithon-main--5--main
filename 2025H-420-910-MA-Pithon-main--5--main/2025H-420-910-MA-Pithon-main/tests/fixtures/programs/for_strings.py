# Test de Alexandrine Dub√©
for i in ["a","b",None, True, 1256, (2,3), [5,6]]:
    print(i) # Output: a b c

for i in ("a","b",None, True, 1256, (2,3), [5,6]):
    print(i) # Output: a b c