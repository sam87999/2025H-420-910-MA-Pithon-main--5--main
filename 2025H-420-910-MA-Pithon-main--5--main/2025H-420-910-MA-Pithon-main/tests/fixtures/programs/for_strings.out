a
b
None
True
1256
(2, 3)
[5, 6]
a
b
None
True
1256
(2, 3)
[5, 6]
